app/assets/stylesheets/templates/_pagination.scss

```
@@ -0,0 +1,25 @@
+.pagination {
```

Sassは僕も100%理解しているわけではないですが、僕が書くなら効果きますかね、、、
これでヤムケンさんの意図することができていれば、こっちの方がいい気がします。

```suggestion
.pagination {
  & li {
    & a {
      border: none;     /*枠線をなくす*/
      color: #696969;   /*文字の色を変える*/
    }
    & a:hover {        
      border-radius: 15px;    /*角を丸くする*/
    }
  }
  & a {
    background: #93c9ff;     /*背景の色を変える*/
    border-radius: 15px;     /*角を丸くする*/
  }
}

---

この書き方になるんですね！どうも一つのpaginationで書け無いな〜と思ってたので、参考になります！
&難しいですね。。修正します！